docs/sdks/client-sdks/javascript/assignments.mdx

```diff
@@ -0,0 +1,411 @@
+---
+title: Assignments
+sidebar_position: 4
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Assignments are the mechanism through which a given [Subject](/sdks/sdk-features/subjects) is assigned to a variation for a feature flag, experiment, or bandit. 
+
+Currently, the Eppo SDK supports the following assignment types:
+
+- String
+- Boolean
+- JSON
+- Numeric
+
+Depending on the values you pass to the getAssignment function, the SDK will return different results based on whether the subject details match the assignment rules you set in the Eppo UI.
+
+This section will cover the different types of assignments that you can make with the Eppo SDK.
+
+### Local Storage
+
+The browser javascript SDK uses browser local storage to store experiment configurations downloaded from Eppo. This allows for quick lookup using one of the get Assignment functions. The configuration data stored contains the experiment key, experiment variation values, traffic allocation, and any allow-list overrides.
+
+## String Assignments
+
+String assignment return a string value that is set as the variation for the experiment. String flags are the most common type of flags. They are useful for both A/B/n tests and advanced targeting use cases.
+
+```js
+import * as EppoSdk from "@eppo/js-client-sdk";
+
+const client = EppoSdk.getInstance();
+
+const flagKey = "flag-key-123";
+const subjectKey = getUserId() || "user-123";
+const defaultAssignment = "version-a"; 
+
+const variant = client.getStringAssignment(
+  flagKey,
+  subjectKey,
+  defaultAssignment
```

Comment:
```suggestion
{},
 defaultAssignment
```

Fairly sure all of these functions need a `subjectAttribute` argument. (Same for the other functions below)